README.md

```diff
@@ -107,42 +111,51 @@
 |-----------------------|------------------------------------------------------------------|-------|
 |CACHE_EXPIRY_IN_SECONDS|The duration in seconds after which cached results will be evicted|300    |
 
-## Trigger Route Toggles
+## Comparison Groups
+
 ### Description
-The toggles to configure which comparators will run.
+
+The following tables contain toggles (for enabling/disabling each comparison) and timer delays (after application startup for each comparison) for their corresponding comparison groups.
```

Comment:
The comparison group that each comparison belongs to is defined by the ComparisonGroup header that has been set in the comparison trigger (which is internal to the application). Therefore:

```markdown
Each comparator belongs to a comparison group. After all comparators in the comparison group have run, 
results produced by each comparator will be published to S3 and a message will be sent to a Kafka topic.

The following tables contain toggles (for enabling/disabling each comparator) 
and timer delays (after application startup for each comparator).
```